### PR TITLE
Create pipeline to publish Swagger Api Parser package

### PR DESCRIPTION
**Pull request by praveenkuttappan**

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
I have disabled few test cases to get pipeline created.

**Comment by praveenkuttappan** [comment]
Fixes #4203 4204

**Comment by ruowan** [comment]
The folder structure has been changed. You are moving the swaggerApiParserTest to parent folder.  Could you explain why we need to do that?  And We need to remove orignal swaggerApiParserTest folder.

**Review by ruowan — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/APIViewWeb.csproj [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <JavaScriptProcessor>..\..\..\..\tools\apiview\parsers\js-api-parser\</JavaScriptProcessor>
     <TypeScriptToolsVersion>latest</TypeScriptToolsVersion>
     <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
+    <SignAssembly>False</SignAssembly>
```
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/APIViewWeb.csproj:16]: Why are we not signing?

**Pull request merged**